e2e/test_build_order.sh

```diff
@@ -50,6 +50,8 @@ if grep -q "skipping building wheels for stevedore" "$log"; then
   pass=false
 fi
 
+APP_ROOT="$(dirname "$(dirname "$(readlink -fm "$0")")")"
```

Comment:
The logs should be created in the $OUTDIR not the APP_ROOT

Reply:
Hmm okay, but we are trying to create `logs` at root level in fromager just like how `work-dir` and `sdists-repo` are created. How do we test this at root level since $OUTDIR is pointing to `e2e-output`. Am I missing something?